.github/snakePipesEnvCI.yml

```diff
@@ -0,0 +1,15 @@
+name: snakePipes_CI
+channels:
+  - conda-forge
+  - bioconda
+dependencies:
+  - python=3.9
+  - snakemake=6.2.1
+  - fuzzywuzzy
+  - mock
+  - sphinx
```

Comment:
I would like to fix the sphinx version to 4.2.0, matching the specs under the docs folder, which I recently updated. Would this be possible?
And `sphinx_rtd_theme==1.0.0` .

Reply:
Done in 870d381